eng/pipelines/coreclr/crossgen2.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
+    jobTemplate: /eng/pipelines/common/build-coreclr-and-libraries-job.yml
```

Comment:
I'm a bit wary of having templates for various combinations... seems like we might start having duplicate boilerplate. I think having something like a "subset matrix" template (like the platform matrix) might be an interesting approach, but I haven't thought it through all the way.

The template in this PR is certainly harmless enough. Maybe it's not even really a concern how well this approach scales, because AFAIK we don't plan on adding tons of these kinds of combinations.

Reply:
I definitely concur this is about to change, Santiago is right now working on unifying library artifacts so that they can be transported across OS / architecture boundaries. I'm not 100% clear on the follow-up plans so I guess we'll just see.